PERFTEST.PS1

```diff
@@ -16,11 +17,12 @@ function input_display {
     Write-Host "============================================"
     Write-Host "$g_path\$scriptName"
     Write-Host " Inputs:"
-    Write-Host "  -Detail = $Detail"
-    Write-Host "  -Diag   = $Diag"
-    Write-Host "  -DestIp = $DestIp"
-    Write-Host "  -SrcIp  = $SrcIp"
-    Write-Host "  -OutDir = $OutDir"
+    Write-Host "  -Detail        = $Detail"
+    Write-Host "  -Diag          = $Diag"
+    Write-Host "  -Configuration = $Configuration"
```

Comment:
Please rename to config throughout.  To seem the variable sizes withing a reasonable range.